src/toil/jobStores/aws/jobStore.py

```diff
@@ -108,6 +151,7 @@ def __init__(self, region, namePrefix, config=None):
         self.filesBucket = None
         self.db = self._connectSimpleDB()
         self.s3 = self._connectS3()
+        self.partSize = partSize
```

Comment:
This should be s3partSize or defaultS3PartSize should be defaultPartSize. I'm ok with either way, as long as it's consistent. defaultPartSize and partSize is probably easier.